.github/workflows/libs-pull-request.yml

```
@@ -3,6 +3,8 @@ run-name: Libs-pull-request
 
 on:
   pull_request:
+    paths:
+      - 'libs/**'
```

You might want to consider including the workflow file itself for any/all workflows. It's not uncommon to introduce changes to the workflow file itself where you'll also want to ensure the workflow itself is evaluated as a consequence of the change.
```suggestion
    paths:
      - 'libs/**'
      - '.github/workflows/libs-pull-request.yml'
```